src/index.njk

```diff
@@ -30,7 +30,7 @@ tags: index-page # Note: This adds a custom class to layout.njk
 
     <div class="section-title">
       <h2>Feature Overview</h2>
-      <p>Explore the wide range of features that Kuadrant offers to enable and empower <b>cluster operators</b> and <b>application developers</b> to work together to securely connect users with their services. Our goal is to empower cluster administrators to deploy and manage gateways at scale to provide application connectivity and give them the confidence to allow application developers to self service knowing that endpoints exposed via these gateways are compliant, secure and resilient. All of this is underpinned by powerful new ingress standard <a href="https://gateway-api.sigs.k8s.io/">Gateway API.</a> and the concepts behind <a href="https://gateway-api.sigs.k8s.io/references/policy-attachment/">policy attachment.</a>
+      <p>Leveraging the emerging ingress standard <a href="https://gateway-api.sigs.k8s.io/">Gateway API </a>, Kuadrant's features enable and empower cluster operators and application developers to work together to securely connect users with their services. Kuadrant empowers cluster administrators to deploy, manage, observe and protect gateways at scale giving them the confidence to allow application developers to self service knowing that endpoints exposed via these gateways are compliant, secure, visible and resilient.</p>
```

Comment:
Kuadrant does not protect gateways, does it? At the data plane, rate limiting and authN/Z policies are meant to protect services, not gateways.

Reply:
well I think that depends? If you have a rate limiting policy targeting a gateway that is setting a infra rate limit is that not protecting the gateway as a piece of that infrastructure

Reply:
Maybe Eguzki’s point is that you always protect the thing 1 or more layers underneath of where you define the rate limit? As in, rate limit policy attached to a gateway protects the routes and the services. Rate limit policy attached to a route protects the service. So to protect a gateway you’d need a RateLimitPolicy at the layer above it?

Reply:
ok new proposed wording https://github.com/Kuadrant/kuadrant.github.io/pull/28

Reply:
> o to protect a gateway you’d need a RateLimitPolicy at the layer above it?

Yes, @thomasmaas !! Thanks for the explanation. I should have been more clear about that. 

PS: Routes are nothing else than configuration of the gateway. RLPs targeting a gateway can be seen as a **default** rate limiting policy for protecting all the services routed through the gateway. RLPs targeting a route (HTTPRoute) can be seen as the rate limiting rules for protecting only the services routed (can be multiple) from that specific route.

Reply:
Agreed but still, 

- they are their own resource type
- the parent of a route doesn't have to be a gateway
- the parent of a route can be multiple gateways